Sources/AsyncHTTPClient/HTTPClient.swift

```diff
@@ -755,22 +760,25 @@ public class HTTPClient {
             self.decompression = decompression
             self.httpVersion = .automatic
             self.networkFrameworkWaitForConnectivity = true
+            self.enableMultipath = enableMultipath
         }
 
         public init(tlsConfiguration: TLSConfiguration? = nil,
                     redirectConfiguration: RedirectConfiguration? = nil,
                     timeout: Timeout = Timeout(),
                     proxy: Proxy? = nil,
                     ignoreUncleanSSLShutdown: Bool = false,
-                    decompression: Decompression = .disabled) {
+                    decompression: Decompression = .disabled,
+                    enableMultipath: Bool = false) {
```

Comment:
Unfortunately, we can't add new parameters to public methods. This will need a new method that has the extra parameter.

Reply:
Does it require a new method, or do you prefer simply having the field enableMPTCP as public in the configuration, allowing to set it freely ?